mne/io/egi/egi.py

```diff
@@ -314,6 +304,8 @@ def __init__(
             raw_extras=[egi_info],
             verbose=verbose,
         )
+        if events_as_annotations:
+            raise NotImplementedError
```

Comment:
wait, so if I'm reading this right, `events_as_annotations=True` only works for `.mff` (not EGI simple binary fmt)?  Docstring needs to say that